src/development/ui/widgets/material3.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Material 3 Components widgets
```

Comment:
Should we just call this page "Material Components widgets", since it's still the same `material` library in Flutter. 

Related, we might want this page to take over the `https://docs.flutter.dev/development/ui/widgets/material` URL and move the legacy M2 page to `https://docs.flutter.dev/development/ui/widgets/material2`. WDYT?

Reply:
Makes sense to me - done (both points).